app/helpers/course_details_helper.rb

```diff
@@ -71,6 +71,19 @@ def language_specialism_options
     [["", ""]] + options
   end
 
+  def provider_options(providers)
+    providers.map do |provider|
+      [
+        provider.name_and_code,
+        provider.id,
+        {
+          "data-append": "<p class=\"govuk-body-s govuk-!-margin-0 govuk-caption-m govuk-hint\">UKPRN: #{provider.ukprn}</p>",
```

Comment:
I think this might need to use `autocomplete__option--hint` in the class? I noticed there's a slight difference in the style between this one:

<img width="733" alt="Screenshot 2023-09-29 at 09 22 33" src="https://github.com/DFE-Digital/register-trainee-teachers/assets/616080/38a965cd-4878-425b-a2fa-53154e7f420f">

and the one used in the schools section for eg:

<img width="659" alt="Screenshot 2023-09-29 at 09 24 50" src="https://github.com/DFE-Digital/register-trainee-teachers/assets/616080/72d8974e-07a0-45d9-ba82-a506d47c34a2">

Reply:
Thanks @darokel good spot, that class does fixes it I think...

![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/450843/709697cf-aee3-4324-9828-5c49ee4ab279)